packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -11,11 +11,17 @@ import { FocusZoneDirection } from './FocusZone.types';
 
 describe('FocusZone', () => {
   let lastFocusedElement: HTMLElement | undefined;
-
+  let tabKeyHit: number;
   function _onFocus(ev: any) {
     lastFocusedElement = ev.target;
   }
 
+  function _onKeyDown(ev: any) {
```

Comment:
use jest.fn() instead of creating your own function. You can access all the arguments passed in after the fact and verify the keycode if you like. It also is cleaner since this code is only used by one test.